h/models/user_deletion.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from sqlalchemy import func
+from sqlalchemy.orm import Mapped, mapped_column
+
+from h.db import Base
+from h.models import helpers
+
+
+class UserDeletion(Base):
+    """A record of a user account that was deleted."""
+
+    __tablename__ = "user_deletion"
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    userid: Mapped[str]
+    requested_at: Mapped[datetime] = mapped_column(
+        server_default=func.now(),  # pylint:disable=not-callable
+    )
+    requested_by: Mapped[str]
+    tag: Mapped[str]
+    created_at: Mapped[datetime]
```

Comment:
created_at is "user.registered_at"  :+1:

Reply:
Yeah. Actually maybe I should rename this to `registered_at` to match, and also add docsttings to these attrs...